config.yml

```diff
@@ -2494,6 +2502,8 @@ nodes:
     fields:
       - name: filepath
         type: string
+        comment: |
+          Represents the name of the file being parsed.
```

Comment:
We need more information than this.

* Is it the name or the path in the filesystem?
* What is the default value?
* How do you set this when you're calling `Prism.parse`?

Reply:
```
require 'prism'

p __FILE__
p Prism.parse("__FILE__")
```
From this, what I understood is that the default value is `""`. If the current file being parsed, will the `filePath` value be the default value?

Reply:
There is documentation on `Prism.parse` that would be relevant here, namely passing the `filepath` keyword. Also `Prism.parse_file` will impact this.